tasks/main.yml

```diff
@@ -17,15 +17,15 @@
   become: true
   template:
     dest: /etc/rsyncd.conf
-    src: etc-rsyncd-conf.j2
+    src: etc/rsyncd.conf.j2
```

Comment:
The rest of our roles use a flat hierarchy for files and templates, so it's preferable to keep this consistent. Renaming `-` to `.` in `-conf.j2` is fine though.

Reply:
Fair enough, consistency is nice. Want me to change this and add it to this pull?

Reply:
Please do.